packages/forms/src/model.ts

```diff
@@ -179,15 +178,13 @@ export abstract class AbstractControl {
   _hasOwnPendingAsyncValidator = false;
 
   /** @internal */
-  // TODO(issue/24571): remove '!'.
-  _pendingTouched!: boolean;
+  _pendingTouched: boolean = false;
```

Comment:
No need for the `boolean` type here as well (it will be inferred by TypeScript):
```suggestion
  _pendingTouched = false;
```